### ISSUE TITLE
In order to assign a task to a courier As a dispatcher I want to be able to move assigned tasks between couriers taskslist

### ISSUE THREAD

**Issue by Atala** [coopcycle/coopcycle-web#3969]
At the moment in the dispatch panel you can not move tasks that are assigned to a courier to another courier directly, because you can only open a tasklist as one.

I do think (to be confirmed by @alexsegura ) that it was only done to simplify stuff in the code, but now with my rework on dragndrop I think it would be quite easy to implement (no promise there ^^)

it was asked by "fortyoneandahalf" [here](https://github.com/orgs/coopcycle/discussions/12#discussioncomment-8110182)
@Paul-Eraman-CoopCycle can you confirm it is useful and if yes prioritize it?

**Comment by Paul-Eraman-CoopCycle** [comment]
I can confirm that this would be very useful, as long as ALSO we can then drag and drop tasks directly from one courier to another instead of unassigning, then reassigning

would this address that as well, or only open two at a time and leave the other behavior unaffected?

**Comment by fortyoneandahalf** [comment]
I specifically asked for drag and drop feature, so yes this should be included

**Comment by Atala** [comment]
yes I was meaning dragndrop which feature were you thinking about paul? you are meaning by right clicking on the task?

**Comment by rayon9coop** [comment]
Drag and drop and right-click: both could also be useful to us. 

I think that drag and drop is foodtech oriented while right clicking is more useful for classic delivery. In our case, I see us instead using the right click function on several tasks. The use case: a courier has a technical incident and I have to move several deliveries assigned to him (but not all) to another courier. In this case, selecting them one by one in a defined order and having the possibility of assigning them in this order to another courier seems easier to me than dragging and dropping deliveries one by one
